components/director/internal/domain/application/service_test.go

```diff
@@ -140,7 +198,11 @@ func TestService_Create(t *testing.T) {
 
 			// then
 			assert.IsType(t, "string", result)
-			assert.Equal(t, testCase.ExpectedErr, err)
+			if err == nil {
```

Comment:
try to simplify it with assert.Equal

Reply:
The basic problem with k8s validation err messages Is that they contain very long message with regex inside.